### DISCUSSION TITLE
Phishing Pages

### DISCUSSION THREAD

**Discussion by selormofmars** [vercel/community#6698] [Help]
### Summary

The following are phishing pages are still live after abuse reports:
http://pancakeswap-claim112.vercel.app/
http://pancakeclick-claimz.vercel.app/

Please deal

### Steps to Reproduce

visit the following websites:
http://pancakeswap-claim112.vercel.app/
http://pancakeclick-claimz.vercel.app/

**Comment by amyegan** [comment]
Thanks for sharing this. Any deployments believed to contain malicious or illegal content should be reported to [abuse@vercel.com](mailto:abuse@vercel.com). This ensures the report is properly tracked in our system and you can receive status updates. I'll let the team know to expect your email.

Here are some additional resources that may help you:
- [Trademark policy](https://vercel.com/legal/trademark-policy)
- [DMCA policy](https://vercel.com/legal/dmca-policy)
- [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support)
